File: 229-majority-element-ii/majority-element-ii.py
class Solution:
    def majorityElement(self, nums: List[int]) -> List[int]:
        dici={}
        for i in range(len(nums)):
            if nums[i] not in dici:
                dici[nums[i]]=1
            else:
                dici[nums[i]]+=1 
        ans=[]
        temp=len(nums)//3
        for i in dici:
            val=dici[i] 
            if val>temp:
                ans.append(i)
        
        return ans                 
        

File: 229-majority-element-ii/majority-element-ii.java
class Solution {
    public List<Integer> majorityElement(int[] nums) {
        HashMap<Integer,Integer> hs=new HashMap<>();
        for(int i=0;i<nums.length;i++){
            hs.put(nums[i],hs.getOrDefault(nums[i],0)+1);
        }
        ArrayList<Integer> li=new ArrayList<>();
        for(Integer key:hs.keySet()){
           int d= hs.get(key);
           if(d>(nums.length)/3){
            li.add(key);
           }
        }
        return li;

    }
}